src/CreateHandler/CreateHandlerCommand.php

```diff
@@ -73,7 +77,7 @@ protected function configure() : void
      */
     protected function execute(InputInterface $input, OutputInterface $output) : int
     {
-        $handler = $input->getArgument('handler');
+        $handler = $input->getArgument($this->handlerArgument);
 
         $output->writeln(sprintf('<info>Creating request handler %s...</info>', $handler));
```

Comment:
Hm, maybe we should then also update message? And help? Hm... (?)

Reply:
It's still creating a _request handler_; the only difference is in the command name and the argument name...

Which is why I'm not particularly convinced this patch is really needed.

@akrabat — what are your thoughts? You can still call them actions even using the `handler:create` command:

```bash
$ ./bin/expressive handler:create "App\Action\HelloWorldAction"
```

The question is really: do we also really want/need an `action:create` command, and, if so, how far should we go towards using "action" terminology in that command? Should the term "request handler" be omitted entirely from help descriptions?

Reply:
It's all down to how accessible this is supposed to be to people who don't know PSR-15 and how "compatible" you want the terminology to be with other frameworks.

Personally, I think that they are actions that happen to implement PSR-15's `RequestHandlerInterface`.

If you want all users to think "Request handler", then this PR isn't needed.

Reply:
Oh and personally,  I'd change message to say "Creating action"

I would have kept the namespace as Action too, so maybe I'm not on the same page though.